scripts/makemcdevsrc.sh

```diff
@@ -5,25 +5,23 @@ set -e
 PS1="$"
 
 basedir="$(cd "$1" && pwd -P)"
-cd "$basedir"
 workdir="$basedir/work"
 minecraftversion=$(cat "$workdir/BuildData/info.json"  | grep minecraftVersion | cut -d '"' -f 4)
 decompiledir="$workdir/Minecraft/$minecraftversion"
-nms="$decompiledir/spigot/net/minecraft/server"
-papernms="Paper-Server/src/main/java/net/minecraft/server"
-mcdevsrc="${decompiledir}/src/net/minecraft/server"
+nms="$decompiledir/spigot/net/minecraft"
+papernms="$basedir/Paper-Server/src/main/java/net/minecraft"
+mcdevsrc="${decompiledir}/src/net/minecraft"
 rm -rf "${mcdevsrc}"
 mkdir -p "${mcdevsrc}"
-find ${nms} -name *.java -print0 | xargs -I\{} -0 cp \{} "${mcdevsrc}/"
+cd "${nms}"
 
-for file in "${nms}/"*
+for file in $(find . -name '*.java')
 do
-    file=${file##*/}
-    # test if in Paper folder - already imported
-    if [ -f "${papernms}/${file}" ]; then
-        # remove from mcdevsrc folder
-        rm -f "${mcdevsrc}/${file}"
+    if [ ! -f "${papernms}/${file}" ]; then
+		cp --parents "${file}" "${mcdevsrc}"
```

Comment:
and in the general scene of BSD being awesome, `--parents` is not standard, likely wanna use dirname to work out the actual tree to create and mkdir -p